front/cypress/e2e/form_builder/elements/number_field.cy.ts

```diff
@@ -29,6 +29,8 @@ describe('Form builder number field', () => {
           title: 'firstPhaseTitle',
           startAt: moment().subtract(9, 'month').format('DD/MM/YYYY'),
           participationMethod: 'native_survey',
+          nativeSurveyButtonMultiloc: { en: 'Take the survey' },
+          nativeSurveyTitleMultiloc: { en: 'Survey' },
```

Comment:
Not for now, but there's a lot of repetition in these e2e tests, so wonder if it would make sense to have additional helper functions such as apiCreateNativeSurveyPhase etc